myhpom/forms/choose_network.py

```diff
@@ -50,7 +41,10 @@ def save(self, *args, **kwargs):
         if not self.cleaned_data['custom_provider']:
             unsaved.cleaned_data = None
 
-        return unsaved.save()
+        saved = unsaved.save()
+        self.instance.health_network_updated = timezone.now()
```

Comment:
Let django do timestamp updates for us by changing this field from `auto_now_add` to `auto_now`.

Reply:
But we don't want the health_network_updated field to update unless the health_network has updated, right?

Reply:
Right - unless I'm mistaken that is what `auto_now` does: https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.DateField.auto_now

Reply:
ohhh....but this isn't the HealthNetwork model - I got myself confused! Disregard!

Reply:
yeah, that's the issue.